modules/account-lib/src/coin/hbar/transferBuilder.ts

```diff
@@ -0,0 +1,91 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
+import Long from 'long';
+import { AccountId } from '@hashgraph/sdk';
+import { proto } from '../../../resources/hbar/protobuf/hedera';
+import { NotImplementedError, BuildTransactionError, InvalidParameterValueError } from '../baseCoin/errors';
+import { BaseKey } from '../baseCoin/iface';
+import { TransactionBuilder } from './transactionBuilder';
+import { Transaction } from './transaction';
+import { isValidAddress, isValidAmount } from './utils';
+
+export class TransferBuilder extends TransactionBuilder {
+  private _txBodyData: proto.CryptoTransferTransactionBody;

```

Comment:
same as wallet init builder

Reply:
In WalletInit is CryptoCreateTransactionBody, but all of the other fields can be moved to the base constructor.